mobify/platform-scaffold
# [Refactor] Transition from urlToPathkeys -> product IDs for Products & Category keys

# [Refactor] Transition from urlToPathkeys -> product IDs for Products & Category keys

 **JIRA**: https://mobify.atlassian.net/browse/WEBDATA-112
**Linked PRs**: https://github.com/mobify/platform-scaffold/pull/704 and https://github.com/mobify/platform-scaffold/pull/733

## Changes
- Transition from urlToPathkeys -> product IDs across both connectors

## How to test-drive this PR
- Tests on both Connectors
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Ensure things look good in the Redux store in terms of product-IDs as keys (under categories, cart, products, integrationManager -> formInfo)
- Refresh on PDP - ensure page still works landing directly on PDP
- Try PLP -> PDP flows
